### PR TITLE
8799 show documentation

### PR DESCRIPTION
**Pull request by arisah-ijeoma**

### CODE REVIEW

File: app/controllers/admin/documentations_controller.rb [hunk 1]
```diff
@@ -24,11 +26,6 @@ def create
     end
   end
 
-  def edit
```
Comment (meltheadorable) [inline app/controllers/admin/documentations_controller.rb]: doesn't this break the edit page?

File: app/policies/documentation_policy.rb [hunk 2]
```diff
@@ -11,4 +11,8 @@ def edit?
   def update?
     create?
   end
+
+  def show?
+    create?
```
Comment (meltheadorable) [inline app/policies/documentation_policy.rb]: this is probably the wrong policy for this, but i'll make a separate story to fix the policies

File: app/controllers/admin/documentations_controller.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 class Admin::DocumentationsController < Admin::AdminController
   include TranslationSaveable
 
+  before_action :find_documentation, only: [:edit, :show]
```
Comment (meltheadorable) [inline app/controllers/admin/documentations_controller.rb]: `update` should probably be in this list too

File: app/views/admin/documentations/show.html.slim [hunk 4]
```diff
@@ -0,0 +1,5 @@
+- content_for(:title, @documentation.page_title)
+
+section.documentation
+  div.documentation-content
+    = simple_format(@documentation.page_content)
```
Comment (meltheadorable) [inline app/views/admin/documentations/show.html.slim]: Great!

File: app/controllers/admin/documentations_controller.rb (hunk 3 above)
Comment (meltheadorable) [reply app/controllers/admin/documentations_controller.rb]: make sure you remove the other find method from update

File: spec/features/admin/documentation_spec.rb [hunk 5]
```diff
@@ -7,7 +7,9 @@
   before do
     login_as user
     doc.summary_content = 'original summary content'
+    doc.page_title = 'original page title'
```
Comment (meltheadorable) [inline spec/features/admin/documentation_spec.rb]: could you refactor these to use a `let!` for creating the doc?

File: app/controllers/admin/documentations_controller.rb (hunk 3 above)
Comment (arisah-ijeoma) [reply app/controllers/admin/documentations_controller.rb]: for some reason, this was breaking the page
will take another look

File: app/controllers/admin/documentations_controller.rb (hunk 1 above)
Comment (arisah-ijeoma) [reply app/controllers/admin/documentations_controller.rb]: nope

File: app/policies/documentation_policy.rb (hunk 2 above)
Comment (arisah-ijeoma) [reply app/policies/documentation_policy.rb]: 👍

File: spec/features/admin/documentation_spec.rb (hunk 5 above)
Comment (arisah-ijeoma) [reply spec/features/admin/documentation_spec.rb]: 👍

File: app/controllers/admin/documentations_controller.rb (hunk 3 above)
Comment (meltheadorable) [reply app/controllers/admin/documentations_controller.rb]: maybe try doing `find(params[:html_identifier])` for all of them instead then, but they should all share the same find action one way or another.

**Pull request merged**